pose-detection/src/movenet/detector.ts

```diff
@@ -63,6 +63,14 @@ export class MoveNetDetector extends BasePoseDetector {
     } else if (config.modelType === SINGLEPOSE_THUNDER) {
       this.modelInputResolution.width = MOVENET_SINGLEPOSE_THUNDER_RESOLUTION;
       this.modelInputResolution.height = MOVENET_SINGLEPOSE_THUNDER_RESOLUTION;
+    } else {
+      if (config.modelUrl.includes("singlepose/lightning")) {
```

Comment:
We could also ask users to supply the model type when they specify a custom URL. But this works for me too.